tareas/Andres_Sealtiel/Tarea_2.jl

```diff
@@ -0,0 +1,619 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .jl
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Julia 1.6.2
+#     language: julia
+#     name: julia-1.6
+# ---
+
+# <div style="text-align: center"> 
+#
+# # Temas Selectos de Física Computacional I
+# ## Tarea 2
+# #### Ramos Arzate Andres & Pichardo Jiménez, Sealtiel ####
+#
+# ### Ejercicio. 1    
+# </div>
+
+"""
+    iterados(f, x₀, c, n)
+Función que calcula los *'n'* iterados de la función *'f(x,c)'*, evaluada en la condición inicial
+*'x₀'* y con constante *'c'*.
+"""
+function iterados(f, x₀, c, n)
+    iters = zeros(n+1)
+    iters[1] = f(x₀, c)
+    for i in 1:n
+        iters[i+1] = f(iters[i], c)
+    end
+    return iters
+end
+
+# ***Avisos***
+#
+# ***1*** *De aquí en adelante se verán varias funciones "silenciadas" (i.e. con un ``#`` al inicio para que no se ejecute el código de esa celda), como sub-funciones de las funciones principales que se ejecutan por individual (ya que se iban probando), así como aquellas que te muestran su documentación. Esto para hacer menos extenso y pesado el notebook de lo que ya es y no ser ejecuciones necesarias para los resultados.*
+#
+# ***2*** *Cuando se ejecuten las gráficas, estas ya tiene parámetros tanto de particion, tamaño de intervalo e iteraciones, tal que se buscó mas o menos estuvieran equilibrados entre dar buenos resultados y no tardarse tanto en ejecutar. Por lo que se recomienda al menos no la primera vez cambiar estos valores. Ya si después se quiere probar la funcionalidad con diferentes parámetros adelante, bajo la advertencia que algunas cosas sí que pueden tardar o inclusive trabar (mi compu murió un par de ocasiones :/ ).*
+
+Q(x,c) = x^2 + c
+
+# +
+#iterados(Q, 0, -3/2, 100_000_000)
+
+# +
+#?iterados
+# -
+
+"""
+    suma_logs(f, x₀, c, n)
+Función que da la suma de *'n'* logaritmos cuyos argumentos son respectivamente cada uno de los
+*'n'* iterados de la función *'f'*, evualada en la condición inicial *'x₀'* y con constante
+*'c'*. Tal suma dividida entre *'n'*.
+"""
+function suma_logs(f, x₀, c, n)
+    args = iterados(f, x₀, c, n)
+    suma = log(abs(2args[1]))
```

Comment:
No entiendo por que está este dos.
```suggestion
    suma = log(abs(args[1]))
```
o lo pusieron por alguna razón?

Reply:
Si, ya que la derivada de x^2 es 2x

Reply:
Ok.